contracts/crowdsale/distribution/FinalizableCrowdsale.sol

```diff
@@ -31,8 +31,8 @@ contract FinalizableCrowdsale is TimedCrowdsale {
      * work. Calls the contract's finalization function.
      */
     function finalize() public {
-        require(!_finalized);
-        require(hasClosed());
+        require(!_finalized, "FinalizableCrowdsale: crowdsale is already finalized.");
```

Comment:
I'd try to drop these extra words, the result is not _too_ terse, but we do end up with more concise messages.
```suggestion
        require(!_finalized, "FinalizableCrowdsale: crowdsale is finalized.");
```